config/routes.rb

```diff
@@ -27,6 +27,11 @@
                  constraints: ->(_) { FeatureFlipper.show_education_benefit_form? }
     end
 
+    resource :health_care_application, only: [:create, :index] do
+      post :create
+      get :index
```

Comment:
These two route definitions are redundant with L30, the resource declaration will create these routes